stories/web3.js

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from 'react';
+
+import { useEthereumProvider } from '../src/web3';
```

Comment:
this story should be written in typescript and should use the typescript types everywhere possible. It is a but unfortunately but the storybook build is part of the manual QA process and can be helpful to catch typescript issues in the public type interfaces the package exposes (e.g. accidental breaking change to types).

Reply:
updated!